### ISSUE TITLE
PHP mysqli failed

### ISSUE THREAD

**Issue by ClearMarble** [laradock/laradock#838]
Hi, I'm trying to move from Homestead to Laradock for local dev, and I'm trying to get a Wordpress project running.

I use the plugin Duplicator to migrate/backup Wordpress sites but when trying to clone this site in Laradock it says PHP mysqli extension is not installed.

I've gone into the .env and set "PHP_FPM_INSTALL_MYSQLI" to "true", but it's still showing as not installed. I've tried destroying and rebuilding the containers, switching to PHP 5.6, and doing a fresh install of Laradock. 

Any idea what I might be doing wrong to get this extension working? 

I'm using Docker for Windows native on Win 10 and bringing things up with 'docker-compose up -d nginx mariadb"

Any pointers or advice would be greatly appreciated. Thanks in advance!

**Comment by dtunes** [comment]
Hi.

can you please try the following and report what you get back.

```sh
docker-compose up -d php-fpm

docker-compose exec php-fpm php -i | grep mysqli
```

this is what I get back using PHP_VERSION=70 and PHP_FPM_INSTALL_MYSQLI=true
```sh
docker-compose exec php-fpm php -i | grep mysqli
/usr/local/etc/php/conf.d/docker-php-ext-mysqli.ini,
mysqli
mysqli.allow_local_infile => On => On
mysqli.allow_persistent => On => On
mysqli.default_host => no value => no value
mysqli.default_port => 3306 => 3306
mysqli.default_pw => no value => no value
mysqli.default_socket => no value => no value
mysqli.default_user => no value => no value
mysqli.max_links => Unlimited => Unlimited
mysqli.max_persistent => Unlimited => Unlimited
mysqli.reconnect => Off => Off
mysqli.rollback_on_cached_plink => Off => Off
API Extensions => mysqli,pdo_mysql

**Comment by dtunes** [comment]
Also, make sure you do the following.

```sh
docker-compose build --no-cache php-fpm
```

**Comment by ClearMarble** [comment]
Ahh...I wasn't aware of the `build --no-cache` option. Had no idea that docker-compose cached anything to begin with. 

When I tried your commands initially it returned nothing, but after `docker-compose build --no-cache php-fpm` and trying again I got the same response as you and the plugin's mysqli test started passing.

Thank you so much for the prompt and extremely helpful response, I'm really grateful!

P.S. Upon actually running the Duplicator installer I kept getting a "504 Gateway Timeout" error from Nginx. Eventually got it to complete by adding  `fastcgi_read_timeout 360;` to default.conf (or 'project-n.conf' for multiple projects) in the `location ~ \.php$ {` section right before `include fastcgi_params;`.

Never had to make that adjustment on Homestead, but everything appears to be working smoothly now.
